### DISCUSSION TITLE
How to perform a firmware update?

### DISCUSSION THREAD

**Discussion by brodyhoskins** [zwave-js/zwave-js-ui#2900] [Q&A]
I'm wondering how to perform a firmware update. Mainly, after I select a file and upload it nothing happens. The only indicator that something might be happening is the error stating another update is in progress if I try to update again. The debug log (set to "silly") only shows updates coming from the node in question.

I was able to update the firmware on one device somehow, but after leaving a second one for 24 hours it just shows "sleeping" and it lets me submit another firmware update; so at this point how can I know if the update is actually happening or not?

I try waking the device several times and 99% of the time it shows as "sleeping" when the device is indicating it's awake.

Is there something I'm missing here? I've tried to solve this on my own relentlessly but I'm getting nowhere.

Thanks

**Comment by robertsLando** [comment]
In case of sleeping device you should manually wake up them. Some have a button to do this. If this doesn't work neighter 
Please attach zwave-js [driver logs](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=generating-log-files)

**Reply by brodyhoskins** [reply]
I'm waking up the device (Aeotec MultiSensor 7) but nothing relevant shows in the logs (just the standard value updates from the sensor to the controller).

The sensor is currently in error state so I'll come back with a log once I have free time.

**Reply by robertsLando** [reply]
Ok let me know so
